tests/ui/strings.stderr

```diff
@@ -60,17 +60,17 @@ error: calling `as_bytes()` on a string literal
    |
    = note: `-D clippy::string-lit-as-bytes` implied by `-D warnings`
 
-error: calling `as_bytes()` on a string literal
-  --> $DIR/strings.rs:55:18
+error: calling `as_bytes()` on `include_str!(..)`
+  --> $DIR/strings.rs:57:22
    |
-55 |     let strify = stringify!(foobar).as_bytes();
-   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider using a byte string literal instead: `bstringify!(foobar)`
+57 |     let includestr = include_str!("entry.rs").as_bytes();
+   |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider using `include_bytes!(..)` instead: `include_bytes!("entry.rs")`
 
 error: manual implementation of an assign operation
```

Comment:
Can you add an `#[allow(clippy::assign_op_pattern)]` to suppress this lint. That's not the intent of these lines.

(Not really part of this PR, but imaginary bonus points for you! :tada: )